lib/trino-hive-formats/pom.xml

```diff
@@ -65,6 +65,12 @@
             <artifactId>guava</artifactId>
         </dependency>
 
+        <dependency>
+             <groupId>commons-codec</groupId>
+             <artifactId>commons-codec</artifactId>
+             <version>1.15</version>
```

Comment:
Should we move this to the root `dependencyManagement`? We have this in a few modules, including `trino-main`